stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/UploadedFile.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.client.models
+
+/**
+ * Represents a successfully uploaded file.
+ *
+ * @param file The URL of the uploaded file.
+ * @param thumbUrl The URL of the thumbnail of the uploaded file.
+ * This property is usually reserved for video files.
```

Comment:
Worded it slightly differently to steer people in the direction that we're only using it for videos so far, otherwise taken your suggestion word for word.